buildSrc/src/main/kotlin/klib/director/JsICCleanerTask.kt

```diff
@@ -0,0 +1,26 @@
+package klib.director
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.*
+import org.gradle.work.InputChanges
+import org.jetbrains.kotlin.gradle.targets.js.ir.JsIrBinary
+import java.io.File
+
+abstract class JsICCleanerTask : DefaultTask() {
+    @get:Internal
+    abstract val incrementalCacheDir: Property<File>
+
+    @TaskAction
+    fun clearIncrementalCache(inputChanges: InputChanges) {
+        incrementalCacheDir.get().deleteRecursively()
+    }
+}
+
+class JsICCleanerTaskConfig(private val binary: JsIrBinary) :
+    KlibDirectorTaskConfigAbstract<JsICCleanerTask>() {
+    override fun configActionImpl(task: JsICCleanerTask) {
+        task.outputs.upToDateWhen { false }
```

Comment:
As I remember it is not necessary, task without `outputs` are always not up-to-date (but I can miss something)